src/components/filter-input/FilterInput.tsx

```
@@ -26,14 +38,21 @@ const FilterInput: FC<FilterInputProps> = ({ value, onChange, ...props }) => {
         <ClearIcon color='secondary' />
       </IconButton>
     ) : (
-      <SearchIcon color='primary' />
+      <IconButton
+        data-testid='search-button'
+        onClick={onSearch}
+        sx={{ p: 0 }}
+      >
```

ESLint

[error] Replace `⏎········data-testid='search-button'⏎········onClick={onSearch}⏎········sx={{·p:·0·}}⏎······` with `·data-testid='search-button'·onClick={onSearch}·sx={{·p:·0·}}`

(prettier/prettier)